dev-aws/kafka-shared-msk/account-identity/holders.tf

```
@@ -23,6 +23,8 @@ module "account_identity_update_holders" {
 
 module "account_identity_legacy_account_holder_compaction" {
   source           = "../../../modules/tls-app"
+  consume_topics   = [(kafka_topic.account_identity_legacy_account_events)]
```

This needs to be the .name.
You can also drop the `()`
```suggestion
  consume_topics   = [kafka_topic.account_identity_legacy_account_events.name]
```